TESTING.md

```diff
@@ -85,15 +85,14 @@ To debug tests in Chrome, prepend `BROWSER=Chrome WATCH=1` to any test command.
 See [the apps readme](./apps/README.md) for more details.
 
 ### Dashboard Tests
-`cd dashboard && RAILS_ENV=test bundle exec rails test` will run all of our dashboard Ruby tests. This can take about 15 minutes to run.
+Before running dashboard tests for the first time, run the below commands to seed the required test data
 
-If you get a bunch of complaints about database, like missing tables or how some tables haven't been seeded, here are some things you can try in order from least to most drastic before running your tests again:
+1. `RAILS_ENV=test bundle exec rake seed:secret_pictures seed:secret_words`
```

Comment:
it looks like you are moving this step from a troubleshooting step to a required step. was this step required for you after running rake install and rake build?

Reply:
I am not sure at this time, there were lots of test failures initially that I'm not sure if this helped resolved a class of issues. I ran this as part of the set up though. Don't think it would be too bad to move this to troubleshooting.